packages/server/graphql/types/PollOption.ts

```diff
@@ -0,0 +1,46 @@
+import {GraphQLID, GraphQLList, GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql'
+import {GQLContext} from '../graphql'
+import GraphQLISO8601Type from './GraphQLISO8601Type'
+import Team from './Team'
+import PollOptionId from '../../../client/shared/gqlIds/PollOptionId'
+
+const PollOption = new GraphQLObjectType<any, GQLContext>({
+  name: 'PollOption',
+  description: 'Poll options for a given poll',
+  fields: () => ({
+    id: {
+      type: GraphQLNonNull(GraphQLID),
+      description: 'Poll option id in a format of `pollOption:idGeneratedByDatabase`',
+      resolve: ({id}) => PollOptionId.create(id)
+    },
+    createdAt: {
+      type: GraphQLNonNull(GraphQLISO8601Type),
+      description: 'The timestamp the item was created'
+    },
+    updatedAt: {
+      type: GraphQLNonNull(GraphQLISO8601Type),
+      description: 'The timestamp the item was updated'
+    },
+    pollId: {
+      type: GraphQLNonNull(GraphQLID),
+      description: 'The foreign key of the poll this option belongs to'
+    },
+    poll: {
+      type: GraphQLNonNull(Team),
+      description: 'The poll this option belongs to',
+      resolve: ({pollId}, _args, {dataLoader}) => {
+        return dataLoader.get('polls').load(pollId)
+      }
+    },
+    voteUserIds: {
```

Comment:
i foresee a `voteUsers` array too, but we can hold off until the front-end gets built